config/initializers/inflections.rb

```diff
@@ -31,6 +31,7 @@
   inflect.singular(/Quota$/, "Quota")
   inflect.plural(/quota$/, "quotas")
   inflect.irregular("container_quota", "container_quotas")
+  inflect.irregular("MiddlewareJms", "MiddlewareJms")
```

Comment:
This has been moved to `config/application.rb` however since we swithced from middleware jms to middleware messaging we no longer need to add inflections in this PR. (plural of messaging is [messagings](https://en.wiktionary.org/wiki/messaging))